db.yml

```diff
@@ -33,13 +61,13 @@ modules:
           _sceAppMgrGameDataMount: 0x454EB183
           _sceAppMgrGetAppInfo: 0x6E65D746
           _sceAppMgrGetAppMgrState: 0x55C60D20
-          _sceAppMgrGetAppParam: 0x258C3E3C
           _sceAppMgrGetAppParam2: 0x26D43923
+          _sceAppMgrGetAppParam: 0x258C3E3C
           _sceAppMgrGetBootParam: 0xEEFBC756
           _sceAppMgrGetBudgetInfo: 0xEACF2237
           _sceAppMgrGetCoredumpStateForShell: 0x3DD7E65
-          _sceAppMgrGetCurrentBgmState: 0x81569A58
           _sceAppMgrGetCurrentBgmState2: 0x3F7B9BB5
+          _sceAppMgrGetCurrentBgmState: 0x81569A58
```

Comment:
Please revert this change and the other changes where `function2` was moved before `function`.